WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/POSCardPresentPaymentMessageView.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+
+struct POSCardPresentPaymentMessageViewModel {
```

Comment:
I think we need to iterate on this model (totally fine on a different PR): Going by its properties, we're allowing the view model to be initialized with all properties being nil or empty, however, all its properties are constants so we cannot re-assign them or perform property injection after it has been initialized. 

I think this could indicate that we're trying to reuse it more than we should and some of its properties should be enforced on initialization by removing the Optional and default values. Perhaps even managing all the properties privately/internally only and just exposing an enum with the type of viewmodel we should initialize 🤔